cairo/kakarot-ssj/crates/evm/src/precompiles/modexp.cairo

```diff
@@ -57,69 +56,205 @@ pub impl ModExp of Precompile {
             }
         };
 
+        // We only support inputs up to 48 bytes. We can early return with an empty span if any size
+        // is larger than that.
+        if base_len > MAX_INPUT_BYTE_SIZE
+            || exp_len > MAX_INPUT_BYTE_SIZE
+            || mod_len > MAX_INPUT_BYTE_SIZE {
+            return Result::Err(EVMError::InvalidParameter('input greater than 48 bytes'));
```

Comment:
It should be added somewhere in the documentation that this will raise for input > 48 bytes

Reply:
yes PR on kakarot docs incoming